matlab/src/cpp/arrow/matlab/bit/unpack.cc

```diff
@@ -38,4 +38,13 @@ namespace arrow::matlab::bit {
 
         return unpacked_matlab_logical_Array;
     }
+
+    const uint8_t* unpacked_as_ptr(const ::matlab::data::TypedArray<bool>& unpacked_validity_bitmap) {
```

Comment:
Hmm. I think that we can have better name for this but I don't have any idea...
(For example, I think that using verb is better... `extract_ptr()`? `extract_raw_data()`?)

Reply:
Good point. I like the name `extract_ptr`.

Reply:
Fixed.